modules/05_initialCap/on/preloop.gms

```diff
@@ -19,16 +19,17 @@ s05_inic_switch = 1;
 q05_eedemini(regi,enty)..
   v05_INIdemEn0(regi,enty)
   =e=
+  (
     !! Pathway I: FE to ppfEn.
     sum(fe2ppfEn(enty,in),
       pm_cesdata("2005",regi,in,"quantity")
     + pm_cesdata("2005",regi,in,"offset_quantity")
-  ) * s05_inic_switch
```

Comment:
why is this switch no longer necessary here?

Reply:
It was on every term separately, I just changed the brackets to be around all the terms and added the switch once for all terms